app/controllers/PanDomainAuthActions.scala

```diff
@@ -44,4 +45,6 @@ class PanDomainAuthActions @Inject() (
 
   override lazy val domain: String = conf.getString("panda.domain").get
   override lazy val system: String = "media-atom-maker"
+
+  override def secret: String = conf.getString("secret").get
```

Comment:
There is a `reference.conf` file that serves as a template for the actual config file. It would be nice to add a new line to it saying something like `secret = "some secret"` and a comment to indicate that it's needed.